pkgs/games/multimc/multimc.desktop

```diff
@@ -0,0 +1,18 @@
+[Desktop Entry]
```

Comment:
You can probably use `makeDesktopItem` for this instead.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/make-desktopitem/default.nix